test/functional/admin/generic_editions_controller_test.rb

```diff
@@ -30,4 +30,22 @@ class Admin::GenericEditionsControllerTest < ActionController::TestCase
 
     assert_select ".govuk-error-summary a", text: "Title is already used on GOV.UK. Please create a unique title", href: "#edition_title"
   end
+
+  test "PATCH :update updates first_published_at to nil when previously_published is false" do
```

Comment:
Is the reverse of this already covered somewhere?

i.e. that the `first_published_at` field gets set correctly when the radio value is `"true"`?

Reply:
yea the 4 tests below cover that behaviour. They started failing without it as this defaults to false in the tests.